tests/gateways/test_repodata_gateway.py

```diff
@@ -72,11 +76,11 @@ def test_stale(tmp_path):
     """
     TEST_DATA = "{}"
     cache = RepodataCache(tmp_path / "cacheme", "repodata.json")
-    MOD = "Thu, 26 Jan 2023 19:34:01 GMT"
-    cache.state.mod = MOD
+    arbitrary_thursday = "Thu, 26 Jan 2023 19:34:01 GMT"
```

Comment:
can we use the date time library in python to get a arbitrary thursday..

Reply:
this is here to test that any assignment to `cache.state.mod` saves and restores the value correctly